### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by rafaelgutierrez-clearcover** [gcanti/fp-ts#1940] [General]
looking for tutorials found this article (https://rlee.dev/practical-guide-to-fp-ts-part-5) that says:

![image](https://github.com/gcanti/fp-ts/assets/149708009/ce7e6d0b-7354-4f81-933a-ac6ed7b3f439)

is this true?

**Comment by gcanti** [comment]
If you are starting from scratch, I also recommend using Effect.

**Reply by p3tr2005** [reply]
why??? i've learn fp-ts and i like the Apis now when i see the Effect, the Apis is really different compared with fp-ts
